.ci/validation.sh

```diff
@@ -262,6 +262,21 @@ verify-no-exception-configs)
   exit $fail
   ;;
 
+verify-regexp-id)
+  fail=0
+  for FILE in config/*_checks.xml
+  do
+    a=$(grep -c "<module name=\"Regexp.*" $FILE) || a=0
+    b=$(grep "<module name=\"Regexp" -A 1 $FILE | grep -c "<property name=\"id\"") || b=0
+    if [ ${a} != ${b} ]
+    then
+      echo "Error: $FILE has Regexp modules without id property"
```

Comment:
technically, id property can be anywhere inside "module" tag and script will fail since it checks only next line after module name.
It is better to change message to clearly show that id should be right next after module name

Reply:
You're right. Done!